docs/WritingTests.asciidoc

```diff
@@ -346,7 +346,7 @@ In short, writing multi-machine tests adds a few more layers of complexity:
 3. actual technical realization (i.e. <<Networking.asciidoc#networking,custom networking>>)
 
 ==== Job dependencies
-There are three dependency *types*: `CHAINED`, `DIRECTLY_CHAINED` and `PARALLEL`
+There different dependency *types*, i.a. _chained_ and _parallel_ dependencies.
```

Comment:
missing verb. And please avoid the abbreviation. I don't even know what you mean. There is https://en.wiktionary.org/wiki/i.a. , do you mean that? Again, better avoid

Reply:
https://dict.leo.org/englisch-deutsch/i.a. - but yes, I suppose I should better avoid it.